src/components/AuthenticatedNavBar.tsx

```diff
@@ -2,22 +2,22 @@ import React from "react";
 import { NavLink } from "react-router-dom";
 import { SignOutButton } from "../api";
 
-import "./AuthenticatedNavBar.css";
+import "./AuthenticatedNavBar.scss";
 
 export function AuthenticatedNavBar() {
 	return (
 		<nav className="Nav">
-			<div className="Nav-container">
+			<div className="Nav__container">
```

Comment:
praise: thanks for the example! still being new to it i was lost for a moment but i think i'm starting to see the vision 😂.